src/multi-level-checkbox-editor/node-label.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
```

Comment:
That component looks so specific to your use cases that it does not seem to be the right place here by now. Please consider moving it to your distinct place of usage.

Reply:
i am not sure  can we pass this from blue print ?

Reply:
separated it